skyportal/facility_apis/observation_plan.py

```diff
@@ -23,9 +24,70 @@
 default_filters = cfg['app.observation_plan.default_filters']
 
 
+def combine_healpix_tuples(input_tiles):
+    """
+    Combine (adjacent?) healpix tiles, given as tuples of (lower,upper).
+    Returns a list of tuples that do not overlap.
+    """
+
+    # set upper bound to make sure this algorithm isn't crazy expensive
```

Comment:
Maybe we print something to the logs (or even errors?!) if this range is exceeded?

Reply:
Seems reasonable.